src/auth-service/bin/cronJob.js

```diff
@@ -16,9 +16,16 @@ cron.schedule("0 0 * * *", async () => {
     while (true) {
       const users = await UserModel("airqo")
         .find({
-          lastLogin: {
-            $lt: new Date(Date.now() - inactiveThreshold),
-          },
+          $or: [
+            {
+              lastLogin: {
+                $lt: new Date(Date.now() - inactiveThreshold),
+              },
+            },
+            {
+              lastLogin: null,
```

Comment:
include situation where the user has not yet logged into the system, just update their active status to false too